### PR TITLE
Hover shake

### LINKED ISSUE #19: Transition on hover
- once user hovers over animated Tv click button then the TV will shake, this will acknowledge the user to click this feature.
- When this animation is finished then start again.
- will include a duration of time that shake will occur.
- using translate in @keyframes that will rotate each shake from percentage of 0%-100%.
- fun feature for interactivity.

### PR DESCRIPTION
**Pull request by PJSalter**
- created transition of hover, once user hovers mouse, the animated Tele shakes 📺 
- named the animation tele-shake.
- animation will last for 1 second in duration.
- used infinite so that the animation never stops unless user moves mouse away.
- @keyframes for px sizings and degrees of shake, this was super fun to experiment with 😸 
- had to change padding to 0.9em instead of 1em of the text background inside the tele because every time shake happened the black background would appear below.
- used CSS Variables for text and background colour inside tele.
- Designed and developed description with the teams names and links to there github profiles.
- fun emojis used about movies.
- short description about project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
-# FAC To-Do list Project
+# :cup_with_straw: FAC To-Do list Project :popcorn:
 
-### Product Design:
+## :performing_arts: Binge List :movie_camera:
```
Comment (mariaalouisaa) [inline README.md:3]: Can never have too many emojis 🤪

File: style.css [hunk 2]
```diff
@@ -111,14 +111,38 @@ h1 {
   bottom: 7.7em;
   left: 1.7em;
   width: 3em;
-  color: #EFD30B;
-  background-color: black;
-  padding: 1em;
+  color: var(--mustard-yellow);
+  background-color: var(--onyx-black);
```
Comment (mariaalouisaa) [inline style.css:115]: Thank you for added the CSS variables! 🙏

File: style.css [hunk 3]
```diff
@@ -111,14 +111,38 @@ h1 {
...
+  20% { transform: translate(-3.7px, 0.4px) rotate(1deg); }
+  30% { transform: translate(3.6px, 2.6px) rotate(0deg); }
+  40% { transform: translate(1.6px, -1.5px) rotate(1deg); }
+  50% { transform: translate(-1.9px, 2.5px) rotate(-1deg); }
+  60% { transform: translate(-3.5px, 1.5px) rotate(0deg); }
+  70% { transform: translate(3.5px, 2.2px) rotate(-1deg); }
+  80% { transform: translate(-2px, -2px) rotate(1deg); }
+  90% { transform: translate(1.8px, 2.5px) rotate(0deg); }
+  100% { transform: translate(1.7px, -2.6px) rotate(-1deg); }
+}
```
Comment (mariaalouisaa) [inline style.css:144]: This is awesome!!! 🎆

**Review by mariaalouisaa — COMMENTED** [review]
Looks great!! Amazing work Peter 💫

**Review by mariaalouisaa — APPROVED** [review]

File: style.css (hunk 3 above)
Comment (AsmahanM) [reply style.css:144]: Really liking the changes you made!

**Pull request merged**